cmd/zetaclientd/utils.go

```diff
@@ -199,5 +202,40 @@ func CreateChainObserverMap(
 		}
 	}
 
+	// FIXME_SOLANA: config chain params
```

Comment:
Can we use `TODO` instead of FIXME for consistency?
Also need an issue to track this, and need more details what's need to be done

Reply:
https://github.com/zeta-chain/node/pull/2465/commits/eb814a91d89bee5be48174081af7338f76cb197a